docs/apache-airflow/tutorial_taskflow_api.rst

```diff
@@ -208,6 +208,18 @@ Python version to run your function.
 These two options should allow for far greater flexibility for users who wish to keep their workflows more simple
 and Pythonic.
 
+Using the TaskFlow API with Sensor operators
+--------------------------------------------
+You can apply the ``@task.sensor`` decorator to convert a regular Python function to an instance of the BaseSensorOperator
+class. The Python function implements the poke logic and returns a Boolean value just as the ``poke()`` method in the
+BaseSensorOperator does.
```

Comment:
I think you shoudl mention `PokeReturnValue` here as well. This is a new addition in Airflow 2.3.0 (I see you have it in type, but the doc should be updated as well).

Reply:
Sure!

Reply:
so :) ?

Reply:
Hi @potiuk I have updated the doc. Please have a look again.